CHANGELOG.md

```diff
@@ -2,6 +2,22 @@
 
 ## master
 
+## [v5.4.0][v5.4.0] (April 12, 2022)
```

Comment:
* `v5.4.0` is not defined anywhere
* I think this deserves a major version bump since we added so many new integrations

Reply:
- I Defined it yesterday, but as we did not release it yesterday I had to delete, otherwise it would be available for public use. I redifined it again today.
- As we are not defining new breaking changes, it would make more sense to use `5.4.0` and not like `6.0.0` as we are following semver.

Reply:
* What I meant is that the rendering is broken because the link to `[v5.4.0]` is not defined anywhere in the document
* You are right. Let's stick to semver. I was thinking of https://semver.org/#spec-item-7 where it says we MAY increment MINOR version. I thought it talked about MAJOR version.

Reply:
I can see you fixed the link as I was typing the comment 👍

Reply:
Sorry about the rendering issue, I forgot to add it.